tutorial/LFRic/README.md

```diff
@@ -0,0 +1,90 @@
+# Tutorial on building LFRic code
+
+This tutorial gives an overview of building LFRic code, with PSyclone
+as an integral part of this process. This repository is structured
+into four subdirectories, one with the
+[introduction to LFRic](background) (including the overview of LFRic
+code structure and how PSyclone is used in LFRic) and three examples.
+
+## Examples
+
+* [**Example 1**](example1) shows how to build and use simple LFRic kernels;
+
+* [**Example 2**](example2) shows the use of PSyclone built-ins;
+
+* **Example 3** shows a time-evolution of a field on a planar mesh (more
+  elaborate example of LFRic code).
+
+In order to mimic the LFRic code structure, an example algorithm and
+main program (driver) are provided in all Examples. In LFRic,
+[kernels](background/LFRic_kernel.md) are *invoke*d from
+[algorithms](background/LFRic_intro.md), which are in turn called
+from the top-level driver code (see the [*Separation of concerns* section](
+background/LFRic_intro.md#separation-of-concerns) in the [LFRic intro](
+background/LFRic_intro.md)).
+
+The kernel and algorithm code in the top-level `example` directories
+is incomplete. The completed solutions are provided in the `solutions`
+subdirectories of each Example, together with `Makefile`s to build
+the code.
+
+## LFRic code support
+
+All the examples here use a pared-down version of the LFRic infrastructure
+stored in `src/psyclone/tests/test_files/dynamo0p3/infrastructure`
+directory of PSyclone repository. For more information on LFRic code
+structure and functionality see [*LFRic code structure* section](
+background/LFRic_intro.md#lfric-code-structure) of
+[introduction to LFRic](background/LFRic_intro.md).
+
+### Implementation of PSyclone in examples
+
+The pared-down LFRic infrastructure used in these examples does not
+have the support for distributed memory (done via the [`YAXT` library](
+https://www.dkrz.de/redmine/projects/yaxt) in LFRic). Also, none of
+the [PSyclone transformations](
+https://psyclone.readthedocs.io/en/stable/transformations.html) are
+applied here so the code is generated and run in serial.
+
+## Software environment
+
+* Linux OS or virtual machine (e.g. [Oracle VM Virtual Box](
+  https://www.virtualbox.org/)).
+
+* Fortran compiler supporting [Fortran 2003 standard.](
+  https://gcc.gnu.org/wiki/GFortranStandards#Fortran_2003)
```

Comment:
May require some F2008 features. In particular, when were `submodules` introduced?

Reply:
Mentioned briefly with the appropriate links and clarified that F2008 this is not required for the examples in this tutorial.